functions/airtable.js

```diff
@@ -640,6 +642,37 @@ async function createFinanceTransaction({ direction, platform, amount, name, not
   );
 }
 
+/* VAX SUPPORT */
+
+const vaxSupportBase = airtable.base(functions.config().airtable.vax_support_base_id);
+
+function createVaxSupportVoicemail(phoneNumber, recordingUrl, message) {
+  // TODO: set up polling for vax support inbound? then below applies
+  // NOTE that we set a `null` status for the record, and let the `onNewInbound` poll function set the status
+  const fields = denormalize(
+    {
+      method: 'Phone Call',
+      phoneNumber: phoneNumber,
+      message: message,
+      voicemailRecording: recordingUrl,
+    },
+    INBOUND_SCHEMA
```

Comment:
the vax support and food intake schemas are the same for now, but should we make a new one just in case/for clarity?